### DISCUSSION TITLE
Provide a way to indicate that a data resource was sampled

### DISCUSSION THREAD

**Discussion by rgbkrk** [frictionlessdata/frictionlessdata.io#659] [Ideas] [answered]
We're using data resource now for sending data from a jupyter kernel to a jupyter frontend. In pandas, you can write:

```python
import pandas as pd
# Turn on pandas application/vnd.dataresource+json for use with
# https://github.com/nteract/nteract/issues/1526
pd.options.display.html.table_schema = True
```

and then every time you display a dataframe, it's returning data resource. _However_, we always either do a `head` or `sample` when representing a dataframe like this (if really big). When that happens we'd like to include data about if/how it was sampled.

We're fine doing something in the metadata section of our own payloads for now, however I'm happy if there's something decent that can go on the actual data resource payload.

/cc @rufuspollock @holdenk

**Comment by rufuspollock** [comment]
@rgbkrk I think this is a quite a common use case and nice to record. It would make a nice "pattern" for sure (would not yet be part of the formal spec).

What's the metadata you are using at the moment?

**Comment by rgbkrk** [comment]
We don't have any metadata yet, https://github.com/nteract/improved-spark-viz/issues/6 was the initial discussion for introducing some.

**Comment by rufuspollock** [comment]
@rgbkrk any update - any mini-pattern is very welcome.

**Comment by rgbkrk** [comment, marked as answer]
We've put a Boolean sampled key on the metadata within our display protocol on the IPython / Jupyter side and didn't set anything on the resource package json.

**Comment by rufuspollock** [comment]
@rgbkrk ok - thanks for updating.
